collect_app/src/main/java/org/odk/collect/android/formmanagement/CollectFormEntryControllerFactory.kt

```diff
@@ -46,6 +47,9 @@ class CollectFormEntryControllerFactory(
                     externalDataHandlerPull
                 )
             )
+
+            it.addFunctionHandler(IntersectsFunctionHandler())
```

Comment:
I'm in two minds about this being untested. It can be seen as pure "configuration" from one angle, but also if I delete it then `intersects` won't work. Maybe it's worth replacing one of the `IntersectsFunctionHandler` tests (probably the detects intersection path) with a feature test?

Reply:
> Maybe it's worth replacing one of the IntersectsFunctionHandler tests (probably the detects intersection path) with a feature test?

I think it would make sense.